elementary_neo4j/neo4j_exporter.py

```
@@ -0,0 +1,119 @@
+import json
+import logging
+from pathlib import Path
+from typing import Any, Dict, List, Optional
```

Flake8 (7.3.0)

[error] 'typing.Optional' imported but unused

(F401)